google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageConfiguration.java

```diff
@@ -60,6 +60,12 @@
   public abstract int blockSize();
 
   /**
+   * Returns the number of times we try re-opening a channel if it's closed unexpectedly
+   * while reading.
+   */
+  public abstract int channelReopen();
```

Comment:
The name `channelReopen` in isolation doesn't sound clear; it sounds like a command, but it's actually just an attribute. I would recommend `maxChannelReopens`.